cmd/sonobuoy/app/results.go

```diff
@@ -66,7 +67,11 @@ func NewCmdResults() *cobra.Command {
 		},
 		Args: cobra.ExactArgs(1),
 	}
-	AddPluginFlag(&data.plugin, cmd.Flags())
```

Comment:
Also, FYI, this was inappropriate to use here. The values `--plugin -p` were ok, but the description of the flags meaning was specific to the `sonobuoy images` command.